src/serial/remote/mod.rs

```diff
@@ -112,6 +128,9 @@ pub trait BmdRemoteProtocol
 	fn supported_architectures(&self) -> Result<Option<TargetArchitecture>>;
 	fn supported_families(&self) -> Result<Option<TargetFamily>>;
 	fn get_target_power_state(&self) -> Result<bool>;
+	fn get_target_voltage(&self) -> Result<f32>;
+	fn get_srst_val(&self) -> Result<bool>;
```

Comment:
This needs to be `nrst`, otherwise this is just plain confusing as `srst` means Software Reset in a pile of places, but this refers to the nRST pin state (ARM trying to make it mean System Reset does not help things.. it just overloads the initialism)